server/configuration.go

```diff
@@ -21,6 +21,7 @@ type Configuration struct {
 	BitbucketOrg               string
 	BitbucketOAuthClientID     string
 	BitbucketOAuthClientSecret string
+	BitbucketSelfHostedUrl     string
```

Comment:
It's Go convention to capitalize URL like so:

```suggestion
	BitbucketSelfHostedURL     string
```

Reply:
Oh, thank you for that! Like I said, I'm new to GoLang, despite having strong experience with other languages hehe

Good to know!